### ISSUE TITLE
Possible behave better on another datasets？

### ISSUE THREAD

**Issue by gonghuiyun** [DAA233/generative-inpainting-pytorch#3]
Hi,
I Implement your code on my datasets with mask size is 50%,  l1 loss is 5.6% lower than the paper reported 8.6%.The result seems great. Is it possible or any problem ?

**Comment by DAA233** [comment]
Hi, @gonghuiyun 

I don't know if it can behave better on your dataset. I only trained the model on ImageNet, and the results are a little worse than the results from the author's TF model. It is tolerable since I only trained it once and didn't tune any hyper-parameters.

As the original author mentioned [here](https://github.com/JiahuiYu/generative_inpainting/issues/146), maybe you shouldn't compare with the numbers in the paper directly:
> if you just want to compare your results with ours in your paper, I suggest you to use our model and compute with your own formulations/implementation.

In my view, you should make sure that you use exactly the same datasets, the same masks, and the same evaluation metrics when you compared with the paper method.
